pyope/ope.py

```diff
@@ -6,6 +6,7 @@
 from pyope.errors import InvalidCiphertextError, InvalidRangeLimitsError, OutOfRangeError
 import pyope.stat as stat
 import pyope.util as util
+import base64, os
```

Comment:
Please place these system imports before 3rd party imports (`Crypto.Cipher`, `Crypto.Util`, etc.), and split them into several lines, one package per line. https://www.python.org/dev/peps/pep-0008/#imports